docs/getting-started-guides/vagrant.md

```diff
@@ -24,6 +24,8 @@ export APOLLO_PROVIDER=vagrant
 sh bootstrap/apollo-launch.sh
 ```
 
+Note: You will either have to run the above command as sudo or have your hosts file writeable for the user running the command
```

Comment:
does this doc only apply if using the vagrant-hosts plugin? might want to put a note in there about that if so.